examples/ensemble/plot_bias_variance.py

```diff
@@ -12,14 +12,14 @@
 predictions of the estimator differ from the predictions of the best possible
 estimator for the problem (i.e., the Bayes model). The variance term measures
 the variability of the predictions of the estimator when fit over different
-instances LS of the problem. Finally, the noise measures the irreducible part
-of the error which is due the variability in the data.
+instances (LS) of the problem, where LS are learning samples. Finally, the noise
+measures the irreducible part of the error which is due the variability in the data.
```

Comment:
```suggestion
learning samples (LS), also know as instances of the problem. Finally, the noise
measures the irreducible part of the error which is due the variability in the
data.
```

As _learning samples_ here is being introduced as a synonym for _instances_, I would prefer keeping the acronym next to its definition for clarity.

I also agree with @thomasjpfan about keeping the rest of the file unchanged.

Reply:
Since this is not a standard notation of the literature, I would rather avoid the "also known as" phrasing:


```suggestion
random instances of the same problem. Each problem instance is noted "LS", for
"Learning Sample", in the following. Finally, the noise measures the irreducible part
of the error which is due the variability in the data.
```

Reply:
As stated above, I agree that the remaining changes of this PR (the "LS" => "S" renamings) need to be undone.